### ISSUE TITLE
Feature Request: Historical timezone data

### ISSUE THREAD

**Issue by wclr** [spencermountain/spacetime#88]
So if I see it right this library doesn't use tz database full data that is used by https://momentjs.com/timezone/ and thus can not be used for historical dates, is spacetime intended only for current moment dates conversions?

**Comment by spencermountain** [comment]
yes, that's correct. I think there are some no-longer-used timezones in our list, but yes, we only record their latest offset+dst change.
I thought i had written that somewhere in the docs, but it appears I haven't. will keep this open until i update it.
cheers

**Comment by whitecolor** [comment]
Don't you want to change add this capability (as optional)? This would make spacetime more universal, allowing to use it in projects that work with historical dates.

**Comment by spencermountain** [comment]
yeah, it gets pretty complex, the further-back you go.
it would also dramatically increase the size of the library

**Comment by ryanblock** [comment]
To further @spencermountain’s point, the entire reason we created Spacetime at @smallwins was because date/time libraries with full historical databases were too large and slow and clunky. So this suggestion kind of negates the whole point of the thing. Perhaps check out Moment, @whitecolor?

**Comment by whitecolor** [comment]
Well, probably there could be created a separate package that would hold the database and use `spacetime` as a dependency.

@ryanblock I use moment-timezone, it requires `moment` that I don't use) that is not a huge problem actually, it can be overcome.

I just thought that `spacetime` as the project could have such a capability.

**Comment by spencermountain** [comment]
been thinking about this a little, and it'd be fun to implement in a separate package somehow.

if you run this, you'll see most computers (mine, atleast) have dst-change info back as far as 1900:
```bash
zdump -v /usr/share/zoneinfo/canada/pacific
```
it'd be fun to turn this into a cleaner json diff, and see how small we can make it.

I guess the names/ids of tzs change too, so **stalingrad time in 1992**  would have some weird edges, and things like this.

if anyone wants to work on it with me, holler.

**Comment by Fellach** [comment]
Hi @spencermountain 

What do you mean by _no-longer-used timezones_? All deprecated on the [IANA](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) list? 

I've got an error `Cannot find timezone named: 'US/Pacific'` and not sure is a bug or feature.

Love your work, greetz!

**Comment by spencermountain** [comment]
@Fellach good question. I think we should include deprecated IANA zones. We have most of them now, i believe. Happy to add `'US/Pacific'`, and any others from that list.

I'm working on softening things up in a [seperate lib](https://github.com/spencermountain/spacetime-informal) for reconciling timezones, like `'India'` or '`San Fransisco`'. That can be where we put the long-tail, real weird timezones, going forward.
cheers

**Comment by johnfumaster** [comment]
Yeah, I've also been getting invalid IANA timezone for 'cet'. Strange enough, when I wrote a unit test to cover this, it doesn't complain.  I've also tried putting it through informal.find and still I get this error sometimes.

**Comment by spencermountain** [comment]
agh weird. Yeah, spacetime-informal should be able to convert `'cet'` to `UTC+3`.
I'd really like to know why you're getting such unreliable results!
cheers
